docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -3,8 +3,7 @@ title: Running an HA Teleport cluster using AWS, EKS, and Helm
 description: Install and configure an HA Teleport cluster using an AWS EKS cluster
 ---
 
-In this guide, we'll go through how to set up a High Availability Teleport cluster with multiple replicas in Kubernetes
-using Teleport Helm charts and AWS products (DynamoDB and S3).
+In this guide, we'll go through how to set up a High Availability Teleport cluster using Teleport Helm charts that runs on AWS EKS.
```

Comment:
```suggestion
In this guide, we'll use Teleport Helm charts to set up a High Availability Teleport cluster that runs on AWS EKS.
```

Alternate wording to make it clear that the Teleport cluster will run on EKS, not the helm charts.